src/nft/components/view/NFTNetwork.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ReactNode } from 'react';
+import { useAccount } from 'wagmi';
+import { baseSvg } from '../../../internal/svg/baseSvg';
+import { cn, text } from '../../../styles/theme';
+
+type NFTNetworkProps = {
+  className?: string;
+  label?: ReactNode;
+};
+
+const networkMap = {
+  Base: baseSvg,
+} as Record<string, ReactNode>;
+
+export function NFTNetwork({ className, label = 'Network' }: NFTNetworkProps) {
+  const { chain } = useAccount();
+
+  if (!chain || !networkMap[chain.name]) {
+    return null;
+  }
+
+  return (
+    <div
+      className={cn(
+        'flex items-center justify-between py-1',
+        text.label2,
+        className,
+      )}
+    >
+      <div>{label}</div>
+      <div className="flex items-center gap-2">
+        <div className="h-4 w-4 object-cover">{networkMap[chain.name]}</div>
+        <div>{chain.name}</div>
+      </div>
+    </div>
+  );
```

Comment:
Probably can remove these divs that don't have styling:

```suggestion
return (
  <div
    className={cn(
      'flex items-center justify-between py-1',
      text.label2,
      className,
    )}
  >
    {label}
    <div className="flex items-center gap-2">
      <span className="h-4 w-4 object-cover">{networkMap[chain.name]}</span>
      {chain.name}
    </div>
  </div>
);
```

Reply:
this is possible, but I think its easier to see whats going on using an element as a flex-child vs. a text node.